benchmarks/instruction_counts/definitions/setup.py

```diff
@@ -0,0 +1,57 @@
+"""Define some common setup blocks which benchmarks can reuse."""
```

Comment:
It would make sense to put parser limitations in this file, many people will look at the examples here, but will never know the rules to obey

Reply:
Done.